tests/sys_crypto/tests-crypto-modes-ccm-128.c

```diff
@@ -0,0 +1,2438 @@
+/*
```

Comment:
- This file was generated from NIST SP 800-38C with a helper script which is very nice! Would you mind to share the script for documentation? You could paste the code snipped somewhere here in the PR.
- Introducing CCM mode tests should also go into its own commit.